sections/main-product.liquid

```diff
@@ -359,6 +143,82 @@
+                         {% for product in collections[block.settings.collection].products %}
+                           <li class="glide__slide">
+                               <div class="card card--cross-sell">
+                               <img src="{{ product.featured_image | img_url: '134x134' }}"  alt="{{ product.title }}"/>
+                               <p class="cross-sell-product-title">{{ product.title }}</p>
+                               <div {{ block.shopify_attributes }}>
+                               {%- if product != blank -%}
+                               <product-form class="product-form">
+                                 <div class="product-form__error-message-wrapper" role="alert" hidden>
+                                    {% render 'svg-product-form-error' %}
+                                     <span class="product-form__error-message"></span>
+                                     </div>
+                                     {%- form 'product', product, id: product_form_id, class: 'form', novalidate: 'novalidate', data-type: 'add-to-cart-form' -%}
+                                       <input
+                                       type="hidden"
+                                       name="id"
+                                       value="{{ product.selected_or_first_available_variant.id }}"
+                                       disabled
+                                       >
+                                       <div class="product-form__buttons">
+                                         <button
+                                         type="submit"
+                                         name="add"
+                                       class="product-form__submit button button--full-width {% if block.settings.show_dynamic_checkout %}button--secondary{% else %}button--primary{% endif %} cross-sell-form-button"
+                                       {% if product.selected_or_first_available_variant.available == false %}
+                                       disabled
+                                     {% endif %}
+                                     >
+                                     <span>
+                                       {%- if product.selected_or_first_available_variant.available -%}
+                                       {{ 'products.product.add_to_cart' | t }}
+                                       {%- else -%}
+                                       {{ 'products.product.sold_out' | t }}
+                                     {%- endif -%}
+                                     </span>
+                                     <div class="loading-overlay__spinner hidden">
+                                      {% render 'svg-loading-overlay-spinner' %}
+                                     </div>
+                                       </button>
+                                       {%- if block.settings.show_dynamic_checkout -%}
```

Comment:
I understaned that you copied this verbatiam from the existing add to cart forms, but a danger from this is that you do not understand what everything you're copying is doing.

The dynamic payment buttons will mess with the design if they do appear to users.